experimental/oak_baremetal_loader/src/main.rs

```diff
@@ -254,7 +254,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         }
     });
 
-    let server_future = server::server("127.0.0.1:8000".parse()?, tx);
+    let server_future = server::server("127.0.0.1:8080".parse()?, tx);
```

Comment:
match the linux runtime default

Reply:
This should really come from args. Hard-coding the address and port was a mistake to begin with.

Reply:
I agree, but also this PR is probably already big enough ;)